llama_index/utils.py

```diff
@@ -38,9 +38,9 @@ class GlobalsHelper:
     def tokenizer(self) -> Callable[[str], List]:
         """Get tokenizer."""
         if self._tokenizer is None:
-            # if python version >= 3.9, then use tiktoken
+            # if python version >= 3.8, then use tiktoken
             # else use GPT2TokenizerFast
-            if sys.version_info >= (3, 9):
+            if sys.version_info >= (3, 8):
```

Comment:
Do we need the `else` part anymore, since llama-index supports python3.8+ ? The else statement would never be reached

Reply:
fixed